tardis/io/tests/test_model_reader.py

```diff
@@ -0,0 +1,35 @@
+import tardis
+from tardis.io.model_reader import (read_artis_density,
+read_simple_ascii_abundances)
+
+from astropy import units as u
+import numpy as np
+
+import pytest
+
+import os
+
```

Comment:
They are 3rd party imports. Only imports from core python are system imports. So the preference is like this
system imports like `import os, sys`
*new line*
3rd party imports like `import astropy, pytest` or anything else
*new line*
our own imports like `from tardis import xyz`